src/components/DragDrop/DragDrop.tsx

```diff
@@ -298,7 +305,7 @@ export const DragDrop = ({
               ref={provided.innerRef}
             >
               <div
-                className={styles['drag-drop__inner']}
+                className={innerClassName}
```

Comment:
Another example for need to refactor is this extraneous `div` wrapper, could probably have functionality on parent element directly (difficult to move in this scope)